autopts/wid/csip.py

```diff
@@ -177,6 +177,7 @@ def hdl_wid_20100(params: WIDParams):
 
     btp.gap_conn(addr, addr_type)
     stack.gap.wait_for_connection(timeout=10, addr=addr)
+    btp.gap_pair(addr, addr_type)
```

Comment:
Hmm isn't this race prone if PTS is also initializing security here?

Reply:
Hmm, if pairing is already in progress and then `bt_smp_start_security` (which `gap_pair` effectively calls) returns -EBUSY. 

Not sure what happens here if/when that happens

Reply:
I think currently zephyr's tester returns BTP error and this will blow up (I think)
(I'm fine if that would be ignored though, either here or in tester)

Reply:
It might be an idea to only call if the security-change event is not received on the next line.